online-shop/src/app/store/selectors/product.selectors.ts

```diff
@@ -0,0 +1,15 @@
+import {createSelector} from "@ngrx/store";
+import {IProductState} from "../state/product.state";
+import {IAppState} from "../state/app.state";
+
+const selectProducts = (state: IAppState) => state.products;
+
+export const selectProductList = createSelector(
+  selectProducts,
+  (state: IProductState) => state.products
+);
+
+export const selectSelectedProduct = createSelector(
```

Comment:
Same as before: do not let unused methods in your code. Still, it is strange that you don't use this method. Because if you don't use this selector at all, this means that you can delete selectedProduct from state as well.